benches/benches/itoa.rs

```diff
@@ -383,13 +383,22 @@ macro_rules! benchmark {
     ($( $target:ident )*) => {$(
         #[bench]
         fn $target(b: &mut test::Bencher) {
-            b.iter(|| for n in 0..314 {
-                let _ = candiate::$target(test::black_box(n));
-            })
+            use rand::prelude::*;
+            let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(314159265358979);
+            let v: [usize; 10000] = std::array::from_fn(|_| rng.next_u64() as usize);
```

Comment:
Thanks, this is better! However, this means that different inputs ( `v`s ) will be used for each benchmark, which should be avoided. To ensure consistent inputs, generate the inputs as a singleton like

```rust
static CASES: std::sync::LazyLock<[usize; 10000]> = std::sync::LazyLock::new(|| {
    use rand::prelude::*;
    let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(314159265358979);
    let v: [usize; 10000] = std::array::from_fn(|_| rng.next_u64() as usize);
    v
});
```

Reply:
No, what was I saying...
all `v`s have the same contents because if the same seed